### PR TITLE
feat(comms):  Update cloud services

### PR DESCRIPTION
**Pull request by GordonSmith**
Added:
* filespray
* wsCloud
* wsElk
* wsPackageProcess

Updated:
* wsDFU

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/comms/package.json [hunk 1]
```diff
@@ -39,17 +39,25 @@
...
         "wsdl-store": "node ./lib-cjs/index.js --url=http://localhost:8010/WsStore?wsdl --outDir=./src/services/wsdl",
         "wsdl-topology": "node ./lib-cjs/index.js --url=http://localhost:8010/WsTopology?wsdl --outDir=./src/services/wsdl",
         "wsdl-workunits": "node ./lib-cjs/index.js --url=http://localhost:8010/WsWorkunits?wsdl --outDir=./src/services/wsdl",
-        "wsdl": "npm-run-all --serial compile-util --parallel wsdl-account wsdl-codesign wsdl-dfu wsdl-dfuxref wsdl-machine wsdl-smc wsdl-store wsdl-topology wsdl-workunits"
+        "wsdl-all": "npm-run-all --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-esdlconfig wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-machine wsdl-packageprocess wsdl-smc wsdl-resources wsdl-store wsdl-topology wsdl-workunits",
+        "wsdl": "npm-run-all --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-packageprocess wsdl-smc wsdl-store wsdl-topology wsdl-workunits"
```
Comment (jeclrsg) [inline packages/comms/package.json:60]: Is there a different use for these two?

**Review by jeclrsg — CHANGES_REQUESTED** [review]
@GordonSmith not sure if the enddate thing in logaccess really matters, I just remembered Rodrigo mentioning that a while back

File: packages/comms/package.json (hunk 1 above)
Comment (GordonSmith) [reply packages/comms/package.json:60]: wsdl-all contained some services which were not available on my installation...

File: packages/comms/src/services/wsdl/ws_logaccess/v1/ws_logaccess.ts [hunk 2]
```diff
@@ -6,73 +6,78 @@ type unsignedInt = number;
...
+    RemoteLogManagerConnectionString: string;
+}
 
-    export interface Range {
-        StartDate: dateTime;
-        EndDate?: dateTime;
-    }
+export interface Range {
+    StartDate: dateTime;
+    EndDate: dateTime;
```
[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/comms/src/services/wsdl/ws_logaccess/v1/ws_logaccess.ts:29]: I think I will leave this as is for now (and raise as an issue for Rodrigo if indeed it is one).

File: packages/comms/package.json (hunk 1 above)
Comment (GordonSmith) [reply packages/comms/package.json:60]: See:  https://github.com/hpcc-systems/Visualization/issues/3961

**Pull request merged**